src/SFA.DAS.AANHub.Application.UnitTests/Notifications/Queries/GetNotificationQueryHandlerTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Notifications.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Notifications.Queries;
+public class GetNotificationQueryHandlerTests
```

Comment:
I think test is missing to validate below case.
if ((notification.MemberId != request.RequestedByMemberId)) { return new ValidatedResponse<GetNotificationQueryResult>((GetNotificationQueryResult)null!); }

Reply:
`Handle_MemberIdDoesNotMatch_Return_Null ` covers this scenario